pkgs/tools/misc/cloc/default.nix

```diff
@@ -11,7 +11,9 @@ stdenv.mkDerivation rec {
     sha256 = "1ihma4f6f92jp1mvzr4rjrgyh9m5wzrlxngaxfn7g0a8r2kyi65b";
   };
 
-  sourceRoot = "source/Unix";
+  setSourceRoot = ''
+    sourceRoot=$(echo */Unix)
+  '';
 
```

Comment:
It's been suggested not to use "source" directly - https://github.com/NixOS/nixpkgs/pull/31106#issuecomment-341266441 - hence I changed it as per the comment.